packages/orbit-components/src/Modal/__tests__/index.test.js

```diff
@@ -14,114 +15,101 @@ import Wallet from "../../icons/Wallet";
 import Button from "../../Button";
 import ChevronLeft from "../../icons/ChevronLeft";
 
-// for testing purposes only
 const Div = (props: any) => <div>{props.children}</div>;
+const onClose = jest.fn();
```

Comment:
`onClose` needs to be defined in every `it` block that's using it, otherwise calls are adding up between tests because `onClose` is always the same mock.

But with the way that you're creating a React element outside of tests this isn't possible, so I suggest the following: instead of rendering one modal that contains everything necessary to test anything, render a separate modal in every test that contains only what's necessary for that particular test, see [InputField test](https://github.com/kiwicom/orbit/blob/4325df7e71ede36fdfe76ab8082fb533728a2732/packages/orbit-components/src/InputField/__tests__/index.test.js) as an example.

Reply:
or to keep things clean, a `beforeEach()` could be defined with `onClose.mockClear()`